themes/powerline-plain/powerline-plain.theme.bash

```diff
@@ -9,17 +9,7 @@ VIRTUALENV_THEME_PROMPT_COLOR=35
 
 SCM_NONE_CHAR=""
 SCM_GIT_CHAR="⎇  "
-SCM_GIT_BEHIND_CHAR="↓"
-SCM_GIT_AHEAD_CHAR="↑"
-SCM_GIT_UNTRACKED_CHAR="?:"
-SCM_GIT_UNSTAGED_CHAR="U:"
-SCM_GIT_STAGED_CHAR="S:"
-
-if [[ -z "$THEME_SCM_TAG_PREFIX" ]]; then
-    SCM_TAG_PREFIX="tag > "
-else
-    SCM_TAG_PREFIX="$THEME_SCM_TAG_PREFIX"
-fi
+SCM_GIT_SHOW_DETAILS=true
```

Comment:
Oh!  Gotcha.  Didn't think that one all the way through.

That said, if someone puts the setting before they load bash-it (like many of the settings are) they might get behavior that's not expected.  Probably not a huge deal, but something to keep in mind.